images/single-cloud/oci/sles-byos/15-sp3/image.yaml

```diff
@@ -0,0 +1,8 @@
+include-paths:
+  - sle15/sp3
```

Comment:
`include-paths` should also include `sle15/sp2` and `sle15/sp1` to inherit from older SPs where applicable. Sorry, missed that earlier.